### PR TITLE
Clarify workflow names

### PR DESCRIPTION
**Pull request by brisvag**
# Description

Just some small clarifications and cleanups for workflow titles in the CI.

### CODE REVIEW

**Review by melissawm — APPROVED** [review]

File: .github/workflows/edit_pr_description.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Remove html In PR description
+name: Clean up PR description
 # see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target
```
Comment (lucyleeow) [inline .github/workflows/edit_pr_description.yml]: I think this link is repeated below?

**Review by lucyleeow — APPROVED** [review]
LGTM!

File: .github/workflows/label_and_milesone_checker.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ on:
 jobs:
   check_labels:
     if: (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'ready to merge'))
-    name: Check PR has type label
+    name: Check PR is marked ready-to-merge
```
Comment (jni) [inline .github/workflows/label_and_milesone_checker.yml]: This change is incorrect. The check is below:

 ```
      - name: Check labels
        uses: docker://agilepathway/pull-request-label-checker:latest
        with:
          any_of: bugfix,feature,documentation,performance,enhancement,maintenance
          repo_token: ${{ secrets.GITHUB_TOKEN }}
```

This *only runs* if the PR is marked ready-to-merge. But what it checks is that the PR has an appropriate type label.

**Review by jni — CHANGES_REQUESTED** [review]
https://github.com/napari/napari/pull/7174/files#r1712582988

File: .github/workflows/labeler.yml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # https://github.com/marketplace/actions/labeler
-name: "Pull Request Labeler"
+name: Add labels
 on:
 - pull_request_target
 
 jobs:
-  triage:
+  labeler:
```
Comment (jni) [inline .github/workflows/labeler.yml:7]: Draga and I have been discussing removing this action altogether. I think those labels ("qt" "tests") are pretty useless?

File: .github/workflows/label_and_milesone_checker.yml (hunk 2 above)
Comment (brisvag) [reply .github/workflows/label_and_milesone_checker.yml]: Ah! Ugh, that's why I was always confused by it...

Comment (brisvag) [reply .github/workflows/label_and_milesone_checker.yml]: Updated to maybe clearer?

File: .github/workflows/labeler.yml (hunk 3 above)
Comment (brisvag) [reply .github/workflows/labeler.yml:7]: Maybe useless for release notes and such, but might be useful for searching PRs and similar. I don't think it hurts :shrug:

Comment (jni) [reply .github/workflows/labeler.yml:7]: The question is whether anyone has ever used them to search for PRs. 😂

**Review by jni — APPROVED** [review]

File: .github/workflows/labeler.yml (hunk 3 above)
Comment (lucyleeow) [reply .github/workflows/labeler.yml:7]: I have!

Comment (lucyleeow) [reply .github/workflows/labeler.yml:7]: Well I do this more in other projects, but I think they can be useful when done well!

**Pull request merged**